stories/Components/Layout/FLDefault/FLDefault.stories.tsx

```diff
@@ -0,0 +1,62 @@
+// Create storybook for FLDefault
+import React from "react";
+import { ComponentStory } from "@storybook/react";
+import { FLDefault } from "@Components/Layout";
+import { IFLDefault } from "@Components/Layout/FLDefault/FLDefault.interface";
+
+const story = {
+  title: "Layout/Default",
+  component: FLDefault,
+};
+
+export default story;
+
+const Template: ComponentStory<typeof FLDefault> = (props: IFLDefault) => (
+  <FLDefault {...props} />
+);
```

Comment:
Follow this https://github.com/HZN-one/flex/pull/63/files#r817334135 for consistency